### ISSUE TITLE
Conflicts when trying to install on Meteor 1.5.1

### ISSUE THREAD

**Issue by vbgm** [nathantreid/blaze-modules#3]
```
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Conflict: Constraint minifier-js@1.1.12 is not satisfied by minifier-js 2.1.1.
Constraints on package "minifier-js":
* minifier-js@2.1.1 <- top level
* minifier-js@2.1.1 <- standard-minifier-js 2.1.1
* minifier-js@1.1.12 <- nathantreid:templating-tools 0.0.4 <- nathantreid:caching-html-compiler 0.0.6 <- nathantreid:templating-components 0.0.6 <-
nathantreid:blaze-modules 0.0.6
* minifier-js@1.1.12 <- nathantreid:templating-tools 0.0.4 <- nathantreid:templating-components 0.0.6 <- nathantreid:blaze-modules 0.0.6

```

**Comment by vbgm** [comment]
@nathantreid any ideas how to fix it?

**Comment by akanix42** [comment]
Since the package complaint is about the minifier-js version, hopefully it's just a matter of bumping the version. I'll see if that does the trick.

**Comment by akanix42** [comment]
This has now been fixed in `nathantreid:blaze-modules@0.0.7`.
